arango/client.py

```diff
@@ -113,7 +119,7 @@ def version(self) -> str:
         :return: Client version.
         :rtype: str
         """
-        return version
+        return version("python-arango")  # type: ignore
```

Comment:
A `# type: ignore` was added due to different `mypy` errors thrown for Python 3.7 / Python 3.8 / Python 3.9 builds.

Unfortunately, a `# type: ignore` [cannot be restricted to one specific python version](https://github.com/python/mypy/issues/8823). This results in having to set `warn_unused_ignores` in `pyproject.toml`. 

Will keep an eye out on the mypy issue.

Reply:
No longer relevant as of https://github.com/ArangoDB-Community/python-arango/pull/206/commits/4b3d067e97f715f9afa463622842bfc1ce593716